inductiva/storage/storage.py

```diff
@@ -133,34 +131,27 @@ def _print_contents_table(contents):
 def upload_from_url(
     url: str,
     remote_dir: str,
-    file_name: str = None,
-    unzip: bool = False,
-    overwrite: bool = True,
+    remote_path: str = None,
```

Comment:
What is the difference from remote_dir and remote_path?

Reply:
remote_dir is the the name of the workspace, remote_path is the path inside the workspace. Accepting better argument names

Reply:
So, do you mean if `remote_dir=hello` and `remote_path=world` this would create the following path `paulo/hello/world` or `paulo/workspace/hello/world`?

Reply:
the prefix "workspace" is always included in GCP bucket but the user doesn't need to know. If u were uploading a file test.txt it would be saved in .../workspace/hello/test.txt

Reply:
So what is the remote path for?

Reply:
sorry, i forgot to add the remote_path, it will be  .../workspace/hello/world/test.txt

remote_path is basically used to specify the path inside the workspace (in this case, workspace named hello)

Reply:
I'm a bit confused why would we need both and not just use remote_dir="hello/world"

Reply:
that makes sense, I will implement that change

Reply:
Upon further investigation I think combining `remote_dir` and `remote_path` might be a bit confusing, especially since we don’t want files stored at the root of the workspace. I believe renaming `remote_dir` to `workspace_name` would make it clearer that this argument is defining a workspace folder, while `remote_path` can still specify a location within that workspace.

This way, it’s clear that files go into a workspace folder, and it feels a bit more intuitive. What do you think?

Reply:
Since we now allow storing files in arbitrary paths, rather than just within the workspace folder with specified filenames, I believe it might make sense to use a single `remote_path` argument. However, if we make that change, we need to decide whether to keep the restriction of only allowing files to be stored inside subfolders (e.g., providing `remote_path`="" would not be allowed).

If we lift this restriction, we might need to adjust the list command and modify the storage database table.

Reply:
I implemented those changes, now there is only a single remote_dir. Let me know what you think